api/Pipfile

```diff
@@ -34,14 +34,14 @@ djangorestframework = "~=3.14"
 drf-spectacular = "*"
 elasticsearch-dsl = "~=7.4"
 future = "~=0.18"
-gunicorn = "~=21.2"
 limit = "~=0.2"
 Pillow = "~=10.0"
 psycopg2 = "~=2.9"
 python-decouple = "~=3.8"
 python-xmp-toolkit = "~=2.0"
 sentry-sdk = "~=1.30"
 django-split-settings = "*"
+uvicorn = {extras = ["standard"], version = "*"}
```

Comment:
Sure! After working with pipenv locally a bit more and seeing how it updates dependencies, I don't think these versions do very much for us. It will update any dependency that has any newer version that matches the constraint any time it runs `install` :cry: